### PR TITLE
Update ApiConfig

### PR DESCRIPTION
**Pull request by C0D3-M4513R**
The specification-test repo has quite a few missing and unmapped properties.
So I just took the liberty of removing things, that don't exist anymore and adding new stuff that is new.

Please note, that this pr does not cover everything that was added.
<details><summary>Things that are still missing</summary>
<p>

- Unexpected property at #/constants
- Unexpected property at #/filterMobile
- Unexpected property at #/forceUseLatestWorld
- Unexpected property at #/generateDisconnect
- Unexpected property at #/googleApiClientId
- Unexpected property at #/keyCompileNotifyDelay
- Unexpected property at #/minSupportedClientBuildNumber
- Unexpected property at #/minimumUnityVersionForUploads
- Unexpected property at #/offlineAnalysis
- Unexpected property at #/photonNameserverOverrides
- Unexpected property at #/sandwichVersion
- Unexpected property at #/searchHeaderGeneratorRoutingReference
- Unexpected property at #/sendTargetScroll
- Unexpected property at #/stackUnityCookieTarget
- Unexpected property at #/syncToken
- Unexpected property at #/timekeeping
- Unexpected property at #/websocketMaxFriendsRefreshDelay
- Unexpected property at #/websocketQuickReconnectTime
- Unexpected property at #/websocketReconnectMaxDelay.

</p>
</details>

Especially constants might be of interest to users.

Fixes: #207

### CODE REVIEW

File: openapi/components/schemas/APIConfig.yaml [hunk 1]
```diff
@@ -428,50 +213,23 @@ properties:
     type: string
   hubWorldId:
     $ref: ./WorldID.yaml
+  imageHostUrlList:
+    description: Allowed hosts, where string data can be loaded from.
```
Comment (ariesclark) [inline openapi/components/schemas/APIConfig.yaml:217]: This description isn't quite accurate, we should probably reference the Udon documentation for https://creators.vrchat.com/worlds/udon/image-loading.

File: openapi/components/schemas/APIConfig.yaml [hunk 2]
```diff
@@ -492,6 +250,11 @@ properties:
     description: Server name of the API server currently responding
     minLength: 1
     type: string
+  stringHostUrlList:
+    description: Allowed hosts, where string data can be loaded from.
```
Comment (ariesclark) [inline openapi/components/schemas/APIConfig.yaml:254]: Same as above.
https://creators.vrchat.com/worlds/udon/string-loading

**Review by ariesclark — CHANGES_REQUESTED** [review]

File: openapi/components/schemas/APIConfig.yaml (hunk 1 above)
Comment (C0D3-M4513R) [reply openapi/components/schemas/APIConfig.yaml:248]: What do you want to be there?
something like `see https://creators.vrchat.com/worlds/udon/image-loading/`?

Comment (ariesclark) [reply openapi/components/schemas/APIConfig.yaml:248]: Something like:

A list of explicitly allowed origins that worlds can request images from via the Udon's [VRCImageDownloader#DownloadImage](https://creators.vrchat.com/worlds/udon/image-loading/#downloadimage).

**Review by ariesclark — APPROVED** [review]

**Pull request merged**